sdk/storage/azure-storage-file-datalake/tests/test_directory.py

```diff
@@ -316,12 +315,12 @@ def callback(response):
                 raise ServiceRequestError("network problem")
         acl = 'user::rwx,group::r-x,other::rwx'
 
-        with self.assertRaises(DataLakeAclChangeFailedError) as acl_error:
+        with self.assertRaises(AzureError) as acl_error:
             directory_client.set_access_control_recursive(acl=acl, batch_size=2, max_batches=2,
                                                           raw_response_hook=callback, retry_total=0)
-            self.assertIsNotNone(acl_error.exception.continuation)
-            self.assertEqual(acl_error.exception.message, "network problem")
-            self.assertIsInstance(acl_error.exception.error, ServiceRequestError)
+            self.assertIsNotNone(acl_error.continuation_token)
```

Comment:
We'll also need to bump the min dependency version of Azure Core as soon as we are able to.

Reply:
sounds great, when do we plan to release azure-core?